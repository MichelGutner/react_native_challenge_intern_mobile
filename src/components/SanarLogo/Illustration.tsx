import React, { Component } from 'react';
import { Image, StyleSheet } from 'react-native';

class Illustration extends Component {
  render() {
    return (
      <Image
        source={require('./LogosImage/illustration.png')}
        style={styles.image}
      />
    );
  }
}

export default Illustration;

const styles = StyleSheet.create({
    image: {
        width: 360, 
        height: 202
    }
})
