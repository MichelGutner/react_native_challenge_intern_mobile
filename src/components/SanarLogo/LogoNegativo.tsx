import React, { Component } from 'react';
import { Image, StyleSheet } from 'react-native';

class SanarLogoNegativo extends Component {
  render() {
    return (
      <Image
        source={require('./LogosImage/Logo-negativo.png')}
        style={styles.image}
      />
    );
  }
}

export default SanarLogoNegativo;

const styles = StyleSheet.create({
    image: {
        width: 152, 
        height: 45
    }
})
