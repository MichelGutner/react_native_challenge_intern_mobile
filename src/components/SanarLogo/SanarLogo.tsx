import React, { Component } from 'react';
import { Image } from 'react-native';

class SanarLogo extends Component {
  render() {
    return (
      <Image
        source={require('./logo-sanar.png')}
        style={{ width: 152, height: 45 }}
      />
    );
  }
}

export default SanarLogo;
