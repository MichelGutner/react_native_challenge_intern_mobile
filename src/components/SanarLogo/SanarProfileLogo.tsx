import React, { Component } from 'react';
import { Image, StyleSheet } from 'react-native';

class SanarProfileLogo extends Component {
  render() {
    return (
      <Image
        source={require('./LogosImage/profile.png')}
        style={styles.image}
      />
    );
  }
}

export default SanarProfileLogo;

const styles = StyleSheet.create({
    image: {
        width: 94, 
        height: 94
    }
})