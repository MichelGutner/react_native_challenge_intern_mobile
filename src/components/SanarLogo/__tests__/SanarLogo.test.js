import React from 'react';
import renderer from 'react-test-renderer';
import SanarLogo from '../SanarLogo';

test('renders correctly', () => {
  const tree = renderer.create(<SanarLogo />).toJSON();
  expect(tree).toMatchSnapshot();
});