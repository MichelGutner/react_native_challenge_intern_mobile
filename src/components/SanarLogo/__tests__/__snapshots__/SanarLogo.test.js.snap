// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<Image
  source={
    Object {
      "testUri": "../../../src/components/SanarLogo/LogosImage/logo-sanar.png",
    }
  }
  style={
    Object {
      "height": 45,
      "width": 152,
    }
  }
/>
`;
